spec/features/create_apo_spec.rb

```diff
@@ -24,5 +34,34 @@
 
     # wait for accessioningWF to finish
     reload_page_until_timeout!(text: 'v1 Accessioned', with_reindex: true)
+
+    # now register an object with this apo and verify default rights
+    visit "#{Settings.argo_url}/registration"
+    # fill in registration form
+    select apo_title, from: 'Admin Policy'
+    click_button 'Add Row'
+    td_list = all('td.invalidDisplay')
+    td_list[0].click
+    fill_in '1_source_id', with: source_id
+    td_list[1].click
+    fill_in '1_label', with: object_label
+    find_field('1_label').send_keys :enter
+
+    click_button('Register')
+    # wait for object to be registered
+    find('td[aria-describedby=data_status][title=success]')
+    object_druid = find('td[aria-describedby=data_druid]').text
+    # puts "object_druid: #{object_druid}" # useful for debugging
+
+    visit "#{Settings.argo_url}/view/#{object_druid}"
+
+    # wait for registrationWF to finish and verify default access rights
+    reload_page_until_timeout!(text: 'v1 Registered', with_reindex: true)
+    expect(page).to have_content "Access Rights:\n#{rights.downcase}"
+
+    # these are in the cocina model data, which is hidden by default
+    expect(page).to have_content(:all, terms_of_use)
+    expect(page).to have_content(:all, copyright)
+    # expect(page).to have_content(:all, license_code) # This should be there, but isn't in cocina!
```

Comment:
If you deploy the main branch of DSA, then it does indeed show the license:
<img width="742" alt="Screen Shot 2021-03-22 at 4 20 47 PM" src="https://user-images.githubusercontent.com/92044/112060026-a4850380-8b2a-11eb-91a3-a36eb0c6bbc2.png">

Reply:
Alrighty then...that's a recent change.  Let me update the test to reflect the fact it will be the URI and not the code.  I'll wait for DSA to be deployed to -prod with the dependency updates.

Reply:
i think you could look for "by-nc" as included in the string, if you preferred but whatever.